src/components/BreakoutRoomsEditor/BreakoutRoomsParticipantsEditor.vue

```diff
@@ -294,8 +295,8 @@ export default {
 		},
 
 		roomName(index) {
-			const roomNumber = index + 1
-			return t('spreed', 'Room {roomNumber}', { roomNumber })
+			return this.breakoutRooms[index]?.displayName
+				?? t('spreed', 'Room manual {roomNumber}', { roomNumber: index + 1 })
```

Comment:
```suggestion
				?? t('spreed', 'Room {roomNumber}', { roomNumber: index + 1 })
```

:confused:

Reply:
Bah, missed it :see_no_evil: 
Though should help with debugging)